ci-operator/step-registry/telcov10n/sno-spoke/cluster/verify/telcov10n-sno-spoke-cluster-verify-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ telcov10n Fix user IDs in a container ************"
+[ -e "${HOME}/fix_uid.sh" ] && "${HOME}/fix_uid.sh" || echo "${HOME}/fix_uid.sh was not found" >&2
+
+function set_hub_cluster_kubeconfig {
```

Comment:
I saw this function 2 or 3 times; it could be better to write it once in a shared utils or similar and reuse it. 
WDYT?